sourcegraph/about
# Fix HubSpot form loading

Closes #5290 by updating the HubSpot hook to ensure that forms load on client-side navigation.

## To Test
- Head to the `/code-insights` page and check that the forms load.
- Head to another page (`/pricing` for example)
- Navigate back to the code insights page and ensure that the forms are loaded.
